enhancements/windows-containers/monitoring-windows-nodes.md

```diff
@@ -0,0 +1,144 @@
+---
+title: monitoring-windows-nodes
+authors:
+  - "@VaishnaviHire"
+  - "@ravisantoshgudimetla"
+reviewers:
+  - "@@openshift/openshift-team-windows-containers"
+approvers:
+  - "@sdodson"
+creation-date: 2021-02-08
+last-updated: 2021-02-11
+status: implementable
+---
+
+# Monitoring Windows Nodes
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The intent of this enhancement is to enable monitoring on Windows nodes created
```

Comment:
Monitoring is a broad term, maybe `performance monitoring`? Its good to be specific when first describing the goals, no need to say anything more than `monitoring` after this first use of the term.